### PR TITLE
changelog for 1.16.0

### PR DESCRIPTION
**Pull request by affeldt-aist**
##### Motivation for this change

In preparation for 1.16.0 release.

##### Things done/to do

- [x] added corresponding entries in `CHANGELOG_UNRELEASED.md` (do not edit former entries)
~~- [ ] added corresponding documentation in the headers~~

##### Automatic note to reviewers

Read [this Checklist](https://github.com/math-comp/math-comp/wiki/Checklist-for-following,-reviewing-and-playing-with-a-PR#checklist-for-reviewing-a-pr) and make sure there is a milestone.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,224 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
-Last releases: [[1.15.0] - 2022-06-30](#1150---2022-06-30), [[1.14.0] - 2022-01-19](#1140---2022-01-19), [[1.13.0] - 2021-10-28](#1130---2021-10-28) and [[1.12.0] - 2020-11-26](#1120---2020-11-26).
+Last releases: [[1.16.0] - 2023-01-31](#1160---2023-01-31), [[1.15.0] - 2022-06-30](#1150---2022-06-30), [[1.14.0] - 2022-01-19](#1140---2022-01-19), [[1.13.0] - 2021-10-28](#1130---2021-10-28) and [[1.12.0] - 2020-11-26](#1120---2020-11-26).
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [1.16.0] - 2022-01-31
```
Comment (proux01) [inline CHANGELOG.md:8]:
```suggestion
## [1.16.0] - 2023-01-31
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,10 +1,224 @@
...
+
+This release is compatible with Coq versions 8.13, 8.14, 8.15, 8.16 and 8.17.
+
+The contributors to this version are:
+
+Cyril Cohen, Enrico Tassi, Erik Martin-Dorel, Georges Gonthier,
+Yoshihiro Ishiguro, Julien Puydt, Kazuhiko Sakaguchi, Laurent Théry,
+Mireia G. Bedmar, Pierre-Marie Pédrot, Pierre Roux, Pierre Pomeret, Quentin VERMANDE,
```
Comment (proux01) [inline CHANGELOG.md:16]:
```suggestion
Mireia G. Bedmar, Pierre-Marie Pédrot, Pierre Roux, Pierre Pomeret, Quentin Vermande,
```
(for the sake of uniformity)
BTW, @pPomCo do you prefer "Pierre Pommeret" or "Pierre Pommeret-Coquot"?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,10 +1,224 @@
...
+- in `ssrnum.v`
+  + lemmas `psumr_neq0`, `psumr_neq0P`
+
+- in `ssrint.v`
+  + printing only notation for `x = y :> int`, opening `int_scope` on
+    `x` and `y` to better match the already existing parsing only
+    notation with the introduction of number notations in `ring_scope`
+
+- in `matrix.v`
+  + definitions `Vandrmonde`, `map2_mx`
```
Comment (proux01) [inline CHANGELOG.md:134]: maybe?
```suggestion
  + definitions `Vandermonde`, `map2_mx`
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,10 +1,224 @@
...
+    `map_comp`, `mapK`, `eq_in_map`
+
+- in `ssrbool.v`, `eqtype.v`, `seq.v`, `path.v`, `fintype.v`, `fingraph.v`,
+  `prime.v`, `binomial.v`, `fingraph.v`, `bigop.v`, `ssralg.v`, `ssrnum.v`,
+  `poly.v`, `mxpoly.v`, `action.v`, `perm.v`, `presentation.v`, `abelian.v`,
+  `cyclic.v`, `frobenius.v`, `maximal.v`, `primitive_action.v`, `alt.v`,
+  `burnside_app.v`, `mxrepresentation.v`, `mxabelian.v`, `character.v`,
+  `inertia.v`, `integral_char.v`, `separable.v`, `galois.v`,
+  `algebraics_fundamental.v`, changed `mem XXX` and `[mem XXX]` to `[in XX]`.
+- Regressions: now redundanrt instances of `inE` rewriting `mem A x` to
```
Comment (proux01) [inline CHANGELOG.md:159]:
```suggestion
- Regressions: now redundant instances of `inE` rewriting `mem A x` to
```

**Review by proux01 — APPROVED** [review]
Some bikeshedding

File: CHANGELOG.md (hunk 2 above)
Comment (affeldt-aist) [reply CHANGELOG.md]: I changed to "Pomeret-Coquot". (I chose "Pomeret" at first based on the email address.)

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,10 +1,224 @@
...
+## [1.16.0] - 2023-01-31
+
+This release is compatible with Coq versions 8.13, 8.14, 8.15, 8.16 and 8.17.
+
+The contributors to this version are:
+
+Cyril Cohen, Enrico Tassi, Erik Martin-Dorel, Georges Gonthier,
+Yoshihiro Ishiguro, Julien Puydt, Kazuhiko Sakaguchi, Laurent Théry,
+Mireia G. Bedmar, Pierre-Marie Pédrot, Pierre Roux, Pierre Pomeret-Coquot, Quentin Vermande,
+Reynald Affeldt, Takafumi Saikawa, Tragicus, Yoshihiro Imai
```
Comment (CohenCyril) [inline CHANGELOG.md:17]:
```suggestion
Mireia G. Bedmar, Pierre-Marie Pédrot, Pierre Roux, Pierre Pomeret-Coquot, Quentin Vermande,
Reynald Affeldt, Takafumi Saikawa, Yoshihiro Imai
```
Quentin Vermande and @Tragicus are the same person. We should alias them in the mailmap.

**Pull request merged**